metal/loadbalancers.go

```diff
@@ -388,6 +393,12 @@ func (l *loadBalancers) addService(ctx context.Context, svc *v1.Service, ips []p
 			// if we did not find an IP reserved, create a request
 			klog.V(2).Infof("no IP assignment found for %s, requesting", svcName)
 			// create a request
+			// our logic as to where to create the IP:
+			// 1. if metro is set globally, use it; else
```

Comment:
Perhaps the more granular setting, the service annotation, should be consulted first with the "default" coming from the global settings?

Reply:
Administrators are more likely to have access to the CPEM runtime parameters.
Users are more likely to have access to the service annotation.

Reply:
If the annotation values permitted need to be restricted, we could introduce another global setting to define the permitted values.

Reply:
You mean to use the env var / CLI flag as a _fallback_ rather than _override_? I can see the use case, but that is not how it has been used in the past. In any case, the only ones ever likely to use env var on the actual CPEM will be administrators. If they set it, they _really_ mean to override.

I don't mind considering having another env var for fallback, but this one historically has been the explicit override; I would like to leave it as is.

Reply:
The flags/env vars really are just meant as controlling overrides for development time. They don't have a good production purpose.